fastlane/fastlane
# [env] add architecture type info to the fastlane env

### Checklist

### Motivation and Context


Resolves #19532
### Description


New variable `Architecture` for architecture type in the `fastlane env` (macOS only).
For more information please see #19532

### Testing Steps


run `fastlane env` and check the result - you should be able to find information about your architecture below the swift version (macOS ONLY).
